include/functions.h

```diff
@@ -97,7 +97,7 @@ void FaultDrawer_SetCharPad(s8 padW, s8 padH);
 void FaultDrawer_SetCursor(s32 x, s32 y);
 void FaultDrawer_FillScreen(void);
 void* FaultDrawer_FormatStringFunc(void* arg, const char* str, size_t count);
-void FaultDrawer_VPrintf(const char* str, char* args);
+void FaultDrawer_VPrintf(const char* str, va_list args);
```

Comment:
```suggestion
void FaultDrawer_VPrintf(const char* fmt, va_list ap);
```
Might as well just match it up with the function definition in `fault_drawer.c`. Don't know how I missed this when I PR'd it.